acquire/dynamic/windows/types.py

```diff
@@ -1,12 +1,201 @@
 from __future__ import annotations
 
 import ctypes
-from ctypes.wintypes import BOOL, DWORD, HANDLE, LPWSTR, PHANDLE, PULONG, ULONG, USHORT
+from ctypes.wintypes import (
+    BOOL,
+    DWORD,
+    HANDLE,
+    LPDWORD,
+    LPVOID,
+    LPWSTR,
+    PHANDLE,
+    PULONG,
+    ULONG,
+    USHORT,
+    WCHAR,
+)
+from enum import Enum, IntEnum
 
 PVOID = ctypes.c_void_p
 NTSTATUS = ULONG
 NULL = None
 
+ULONG_PTR = ctypes.c_size_t
+
+DUPLICATE_SAME_ACCESS = 0x00000002
```

Comment:
Placed it in ProcessAccess IntEnum

Reply:
Would it be an issue that it has the same value as `PROCESS_CREATE_THREAD` in that same enum?

Reply:
Well, in this case it does not, since here the enum is used only as a one-way lookup (namely `ProcessAccess.DUPLICATE_SAME_ACCESS` for example). However, it could lead to some confusion/errors if you use it the other way around.

So, do we want to change it now, or will we use the enum only in this particular way?

Reply:
I'd make it a separate enum. I found there is at least another option (even 2 in WIN2000):
```
class DuplicateHandleOptions(IntEnum):
    DUPLICATE_CLOSE_SOURCE = 0x00000001
    DUPLICATE_SAME_ACCESS = 0x00000002
    DUPLICATE_SAME_ATTRIBUTES = 0x00000003
```

The last one, `DUPLICATE_SAME_ATTRIBUTES` I could only find for WIN2000, so I'm not sure it should be included.